code/game/objects/items/devices/pager.dm

```diff
@@ -0,0 +1,147 @@
+var/list/pager_list = list()
+
+/obj/item/device/pager
+	name = "station alerts pager"
+	desc = "Alerts engineers of any triggered station alarms."
+	icon_state = "pager_muted"
+	item_state = "electronic"
+	flags = FPRINT
+	siemens_coefficient = 1
+	force = 5.0
+	w_class = W_CLASS_SMALL
+	throwforce = 5.0
+	throw_range = 15
+	throw_speed = 3
+	starting_materials = list(MAT_IRON = 500, MAT_GLASS = 200)
+	w_type = RECYK_ELECTRONIC
+	melt_temperature = MELTPOINT_SILICON
+	origin_tech = Tc_ENGINEERING + "=3;" + Tc_BLUESPACE + "=2"
+	slot_flags = SLOT_BELT
+	var/muted = TRUE
+	var/list/prefs = list("Power","Fire","Atmosphere")
+	var/last_alert_time = 0
+	var/alert_delay = 10 SECONDS
+	var/last_alert = ""
+
+/obj/item/device/pager/proc/get_pager_list()
+	return pager_list
+
+/obj/item/device/pager/New()
+	..()
+	pager_list = get_pager_list()
```

Comment:
I suspect this part was stolen from SPS code but it has no use here. The reason why the SPS does it like this is to allow subtypes to implement get_sps_list() to be placed in a separate list from the others. You can just reference the one global list you have directly here.

Reply:
I did find some utility in this only because it is a pain to check global var status while debugging. As it's really only used for debugging here, I will remove it.